golem-service/src/config.rs

```diff
@@ -15,6 +16,13 @@ pub struct DbSqliteConfig {
     pub max_connections: u32,
 }
 
+#[derive(Clone, Debug, Deserialize)]
+pub struct WorkerConnectCacheConig {
```

Comment:
Let's call it WorkerExecutorClientCache or something, it's not for the "connect" thing only

Reply:
Sure yea!